assets/src/dashboard/app/api/useCategoriesApi.js

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { useCallback, useEffect, useMemo, useState } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import groupBy from '../../utils/groupBy';
+
+export default function useCategoriesApi(dataAdapter, { wpApi }) {
+  const [categories, setCategories] = useState({});
+  const fetchCategories = useCallback(async () => {
+    try {
+      const response = await dataAdapter.get(wpApi, {
+        parse: false,
```

Comment:
I believe `parse: false` is only used by the story API so we could read the response header tags to get some extra info.  In this case, it looks like you're taking the response and immediately converting it to `json`, so can do it in one step like this:
```
const categoriesJson = await dataAdapter.get(wpApi);
```